taming/models/cond_transformer.py

```diff
@@ -239,7 +239,16 @@ def log_images(self, batch, temperature=None, top_k=None, callback=None, lr_inte
         log["inputs"] = x
         log["reconstructions"] = x_rec
 
-        if self.cond_stage_key != "image":
+        if self.cond_stage_key in ["objects_bbox", "objects_center_points"]:
+            figure_size = (x_rec.shape[2], x_rec.shape[3])
+            dataset = kwargs["pl_module"].trainer.datamodule.datasets["validation"]
+            label_for_category_no = dataset.get_textual_label_for_category_no
+            plotter = dataset.conditional_builders[self.cond_stage_key].plot
+            log["conditioning"] = torch.zeros_like(log["reconstructions"])
+            for i in range(quant_c.shape[0]):
+                log["conditioning"][i] = plotter(quant_c[i], label_for_category_no, figure_size)
+            log["conditioning_rec"] = log["conditioning"]
+        elif self.cond_stage_key != "image":
```

Comment:
@rromb IMHO this change should not break anything, but it can't hurt that you double-check

Reply:
I still need to check the `plotter`, but the general logic looks good and shouldn't break anything :)

Reply:
looks good!